src/lib/buildCustomTypeDataType.ts

```diff
@@ -1,4 +1,4 @@
-import { CustomTypeModel } from "@prismicio/client";
+import type { CustomTypeModel } from "@prismicio/client";
```

Comment:
Maybe we could add on this repo (or on the template also) https://typescript-eslint.io/rules/consistent-type-imports to prevent those mistakes? (since they seem to be critical here)

Reply:
It's not critical here; I added them just to be explicit and consistent since some of the files already used `import type`. Bundling `@prismicio/client` was the critical change in this PR.

Reply:
Ohhh, ok got it!